force-app/main/default/lwc/loadSpinner/loadSpinner.html

```diff
@@ -0,0 +1,12 @@
+<template>
+  <div class="demo-only" style="height: 6rem; position: relative">
```

Comment:
in case you didn't see it, here's the OOTB spinner - https://developer.salesforce.com/docs/component-library/bundle/lightning-spinner/example